CVLByExample/ConfInheritance/invalid_base.conf

```
@@ -0,0 +1,11 @@
+{
+  "files": [
+    "MainContract.sol",
+  ],
+  "verify": "MainContract:MainContract.spec",
```

```suggestion
  "verify": "MainContract:MainSpec.spec",
```
Right now if we run the spec we fail with the error:
`attribute/flag 'verify': file MainContract.spec not found`